packages/adminrouter/extra/src/includes/http/common.conf

```diff
@@ -1,9 +1,17 @@
-# Enable VTS module for metrics
-# Set the shared memory zone size to 32MB to be sure to
-# contain all metrics in the VTS module configuration.
-# If the VTS module configuration enables dynamic metric
-# keys this needs to be increased to accommodate for that.
-vhost_traffic_status_zone shared:vhost_traffic_status:32m;
+# Enable VTS module for metrics.
+# The shared memory zone size must be sufficiently large to ensure that all
+# metrics that could potentially be generated by the VTS module's
+# vhost_traffic_status_filter_by_set_key directive can be stored. The metrics
+# data scales with the number of metric nodes. A metric node in our case is a
+# set of data about one detail, such as the number of requests made with a
+# particular client, or the number of responses with a particular status code,
+# or the number of URIs requested from Admin Router. In theory, the number of
+# URIs can be infinitely large. If the shared memory zone is near 86% of the
```

Comment:
Ideally let's get a reference to 86% - how this was determined and ideally a link to code in another tool which chooses it.

Reply:
If we can't, let's at least put a comment how we determined this.
Why only 86%!